src/app/pages/registration/components/registration-form/registration-form.component.html

```diff
@@ -59,25 +59,27 @@ <h2>
                 controlName="termsAndConditions"
                 [errorMessages]="{ pattern: 'registration.tac.error.tip' }"
               >
-                <span>{{ 'registration.tac_privacy_policy.begin.text' | translate }} </span>
-                <ish-modal-dialog-link
-                  linkText="registration.tac.link"
-                  [options]="{ titleText: 'checkout.termsandconditions.details.title' | translate, size: 'lg' }"
-                >
+                <span
+                  [ishServerHtml]="'registration.tac_privacy_policy.label' | translate"
+                  [callbacks]="{
+                    callbackTAC: showModalDialog(modalDialogTAC),
+                    callbackPP: showModalDialog(modalDialogPP)
```

Comment:
this would also work:
```
                  [callbacks]="{
                    callbackTAC: modalDialogTAC.show,
                    callbackPP: modalDialogPP.show
                  }"

```

That way the method in the component file would not be needed.

Reply:
I seem to have mis-tested. :sweat_smile: